src/lists/lists.test.ts

```diff
@@ -0,0 +1,23 @@
+import { clearItemsFromMovieList } from "./lists";
+import { getMovieListDetails } from "./lists";
+
+describe("Test for Lists  API ", () => {
+  test("Check Status code.", async () => {
+    const { status } = await clearItemsFromMovieList();
+    expect(status).toBe(201);
+    
```

Comment:
1. First get any item in the movie list and assign it to a variable. <br>
2. Clear the list. <br>
3. Either check the item doesn't exist in the list anymore or check movie list length is equal to zero.